src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -102,13 +102,17 @@ function PurchaseOrderUtility({ creating }) {
             field: 'dateRequested',
             headerName: 'Request Date',
             width: 200,
-            type: 'date'
-        },
-        {
+            renderCell: (params) => {
```

Comment:
same outcome but I'd say valueFormatter is a bit smoother than re-rendering the whole cell.